sample/test/fails/test.yaml

```diff
@@ -9,19 +9,29 @@ test:
         - name: fail-output-1
           exec: test-fail-output-1
           harness: test-fail-output-1.py
+          targets:
+              exclude: [armv7a9-zynq7000-emu]
```

Comment:
If You need to exclude the new target in almost all tests **it shouldn't be put in DEFAULT_TARGETS** for now.

Reply:
That's right, including it in psh for psh tests would be better